DER/DerivativesContracts/Forwards.rdf

```diff
@@ -150,86 +206,178 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-der-der-fwd;ForwardDeliveryCommitment">
 		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;DeliveryTerms"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-txn-ev;madeBy"/>
+				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasAdditionalCosts"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasForwardUnderlierPrice"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;appliesTo"/>
 				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;OptionUnderlyingSeller"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">forward delivery commitment</rdfs:label>
 		<skos:definition xml:lang="en">commitment to deliver something in the future, at an agreed date or dates and under agreed terms</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">In an Option, there is still a commitment on one side to deliver in the event that the other side chooses to exercise the option.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Typically this is a commitment to deliver cash, but it could also involve some other underlying asset. In an Option, there is still a commitment on one side to deliver in the event that the other side chooses to exercise the option.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-der-der-fwd;ForwardTransaction">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;SecuritiesTransaction"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;embodies"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-fwd;Forward"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">forward transaction</rdfs:label>
 		<skos:definition xml:lang="en">A transaction in which one party commits at the present to delivery of something in the future.</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-der-fwd;ForwardValuationTermsSet">
+	<owl:Class rdf:about="&fibo-der-der-fwd;ForwardValuationTerms">
```

Comment:
We've actually done this everywhere throughout FIBO - it's not a set of terms in many cases.  I would rather not go back to TermsSet frankly, so I only updated it to be in line with what we did elsewhere.  It might be that Term is more appropriate, or something else, so I'll do some homework with Rob, who is a practicing attorney, but would need to do something that is 'the same' across all of these things.